cypress/e2e/swap/uniswapx.test.ts

```diff
@@ -1,32 +1,55 @@
-import { DAI, USDC_MAINNET } from '../../../src/constants/tokens'
+import { ChainId, CurrencyAmount } from '@uniswap/sdk-core'
+
+import { DAI, nativeOnChain, USDC_MAINNET } from '../../../src/constants/tokens'
 import { getTestSelector } from '../../utils'
 
 const QuoteEndpoint = 'https://api.uniswap.org/v2/quote'
 const QuoteWhereUniswapXIsBetter = 'uniswapx/quote1.json'
+const QuoteWithEthInput = 'uniswapx/quote2.json'
+
+const OrderSubmissionEndpoint = 'https://api.uniswap.org/v2/order'
+const OrderResponse = 'uniswapx/orderResponse.json'
+
+const OrderStatusEndpoint =
+  'https://api.uniswap.org/v2/orders?swapper=0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266&orderHashes=0xa9dd6f05ad6d6c79bee654c31ede4d0d2392862711be0f3bc4a9124af24a6a19'
+const OpenStatusResponse = 'uniswapx/openStatusResponse.json'
+const FilledStatusResponse = 'uniswapx/filledStatusResponse.json'
+const ExpiredStatusResponse = 'uniswapx/expiredStatusResponse.json'
+const TransactionReceipt = 'uniswapx/fillTransactionReceipt.json'
```

Comment:
nit: I think you could inline all of these fixture paths. they're already self-documented from the filenames.